hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/configuration/DFSDeltaConfig.java

```diff
@@ -36,15 +36,22 @@
   private final Long maxFileSize;
   // The current batch id
   private Integer batchId;
+  // Paralleism to use when generating input data
+  int inputParallelism;
```

Comment:
can you explicitly add scope "private" ?

Reply:
Done.